scripts/install-dotnet-macos.sh

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
What's the reason behind this empty line (also in install-windows) :)

Reply:
Replaced with shebang